Integrations/SplunkPy/SplunkPy.py

```diff
@@ -489,6 +489,60 @@ def splunk_submit_event_command():
         demisto.results('Event was created in Splunk index: ' + r.name)
 
 
+def splunk_submit_event_hec(hec_token, baseurl, event, fields, host, index, source_type, source, time_):
+
+    if hec_token is None:
+        raise Exception('The HEC Token was not provided')
+
+    args = {}
+    args['event'] = event
+    if fields:
+        args['fields'] = {
+            'fields': fields
+        }
+    if host:
+        args['host'] = host
+    if index:
+        args['index'] = index
+    if source_type:
+        args['sourcetype'] = source_type
+    if source:
+        args['source'] = source
+    if time_:
+        args['time'] = time_
+
+    headers = {
+        'Authorization': 'Splunk %s' % hec_token,
```

Comment:
Is there a special reason to use `%` and not `.format()` here?

Reply:
No special reason.. it's just the way it was written in the integration so far.
Will change that